fsspec/implementations/reference.py

```diff
@@ -213,27 +295,63 @@ def items(self):
         return RefsItemsView(self)
 
     def __getitem__(self, key):
-        if key in self._items:
-            val = self._items[key]
-            if val is None:
-                raise KeyError
-        if key in self.zmetadata:
-            # spec requires bytes even if we already decoded the metadata
-            return json.dumps(self.zmetadata[key]).encode()
```

Comment:
Why was this portion deleted? I presume there will be logic added back later.

Reply:
The same logic is in _load_one_key